app/overrides/spree/admin/general_settings/edit/legal_settings.html.haml.deface

```diff
@@ -0,0 +1,13 @@
+/ insert_after "fieldset.security"
+
+%fieldset.legal.no-border-bottom
+  %legend{:align => "center"}= t(:legal_settings)
```

Comment:
Have you tried to use lazy lookup in deface files? It would be good to know if that works. I would just do `t('.legal_settings')` and then check the error message to see where it expects the key to be. We can then move all these strings into the right scope.

Reply:
yes, it works. Thanks for the hint!
I have added it to "add translations' here: https://github.com/openfoodfoundation/openfoodnetwork/wiki/Internationalisation-(i18n)#development

I am now using lazy loading for these translations of the legal settings 👍